external/uapaotredist/project.json.template

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "frameworks": {
+    "uap10.1": {
+      "dependencies": {
+        "System.Diagnostics.Tracing": "4.4.0-beta-24906-01",
```

Comment:
I suspect that that build was just redistributing an early binary itself (via harvesting).  It may be worthwhile to check the package content and determine if we could use a stable version since that'll reduce the package versions downloaded.